content/blog/2015-06-03-service-discovery-introduction.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Service Discovery Introduction
+created_at: 2015-06-02
+kind: article
+author: Fabian Reinartz
+---
+
+This week we released Prometheus v0.14.0 — a version with many long-awaited additions
+and improvements.
+
+On the user side, new mechanisms for service discovery aim to make it easier to
+dynamically update the targets Prometheus scrapes. In addition to DNS-SRV records,
+[Consul](https://www.consul.io) is now supported out-of-the-box and a file-based interface
+allows you to connect your own discovery mechanism.
+Over time, we plan to extend Prometheus with native support for other common ways of doing
```

Comment:
"Over time, we plan to add other common service discovery mechanisms to Prometheus." sounds simpler